docs/auth_permissions.md

```diff
@@ -66,31 +72,17 @@ If a user is a member of multiple groups, the groups permission policies will be
 Let's look at an example:
 
 ```python
-{
-  "entities": {
-    "entity_ids": {
-      "light.kitchen": True
-    }
-  }
-}
+{"entities": {"entity_ids": {"light.kitchen": True}}}
```

Comment:
I don't think that this is an improvement.

Reply:
Like, it hurts readability ?

Reply:
It's how black does it, and we're deferring to it in the codebase, too. Would you prefer manual formatting instead here?

Reply:
Yeah. Auth is already difficult to grasp, wouldn't want to make it more difficult (although we don't use it)

Reply:
When coding, I idents which help me to visualize the structure. Black is going to reformat it later and that's fine.